.github/actions/redhat-opdev-preflight-action/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Fail fast.
+set -euo pipefail
+
+# Login.
+echo "$INPUT_PASSWORD"  | docker login scan.connect.redhat.com -u "${INPUT_USERNAME}" --password-stdin
```

Comment:
Can add an explicit check for `$INPUT_PASSWORD` and `$INPUT_USERNAME` if they're not empty?

Reply:
Maybe in opdev action?

Reply:
Good idea ( you mean marking those as required in action? 👍 ) but I think that it won't hurt to put it in both places.

Reply:
I mean adding the validation in the script in the other repo and leaving it as is in this PR.

Reply:
Let's add in both places. It's an easy tweak and we'll already have the "final version" in place before the switch to action from the other repo.

Reply:
Please see the latest change.